src/__tests__/screen.js

```diff
@@ -1,9 +1,63 @@
 import {renderIntoDocument} from './helpers/test-utils'
 import {screen} from '..'
 
+beforeEach(() => {
+  jest.spyOn(console, 'log').mockImplementation(() => {})
+})
+
+afterEach(() => {
+  console.log.mockRestore()
+})
+
 test('exposes queries that are attached to document.body', async () => {
   renderIntoDocument(`<div>hello world</div>`)
   screen.getByText(/hello world/i)
   await screen.findByText(/hello world/i)
   expect(screen.queryByText(/hello world/i)).not.toBeNull()
 })
+
+test('exposes debug method', () => {
+  renderIntoDocument(
+    `<button>test</button><span>multi-test</span><div>multi-test</div>`,
+  )
+  expect(screen.debug).toBeInstanceOf(Function)
```

Comment:
I don't think we don't need this assertion because we make it implicitly by calling it. Implicit assertions aren't always the best, but when it's something as simple as what type something is I don't think it's all that useful to have.